tests/backup/backup_test.go

```diff
@@ -2661,6 +2661,263 @@ var _ = Describe("{BackupAlternatingBetweenLockedAndUnlockedBuckets}", func() {
+				bkpNamespaces = append(bkpNamespaces, namespace)
+			}
+		}
+	})
+	It("Validate after toggling the access, user can perform operation on new backup", func() {
+		providers := getProviders()
+		Step("Validate applications", func() {
+			log.InfoD("Validate applications")
+			ValidateApplications(contexts)
+		})
+		Step("Create User", func() {
+			err := backup.AddUser(userName, firstName, lastName, email, password)
+			log.FailOnError(err, "Failed to create user - %s", userName)
+
+		})
+		Step("Adding Credentials and Registering Backup Location", func() {
+			log.InfoD("Creating cloud credentials and backup location")
+			for _, provider := range providers {
+				cloudCredUID = uuid.New()
+				CloudCredUidList = append(CloudCredUidList, cloudCredUID)
+				backupLocationUID = uuid.New()
+				credName = fmt.Sprintf("autogenerated-cred-%v", time.Now().Unix())
+				CreateCloudCredential(provider, credName, cloudCredUID, orgID)
+				log.InfoD("Created Cloud Credentials with name - %s", credName)
+				time.Sleep(time.Minute * 3)
+				backupLocationName = fmt.Sprintf("autogenerated-backup-location-%v", time.Now().Unix())
+				newBackupLocationMap[backupLocationUID] = backupLocationName
+				err := CreateBackupLocation(provider, backupLocationName, backupLocationUID, credName, cloudCredUID, getGlobalBucketName(provider), orgID, "")
+				log.FailOnError(err, "Creating Backuplocation %v", backupLocationName)
+				log.InfoD("Created Backup Location with name - %s", backupLocationName)
+			}
+		})
+		Step("Register source and destination cluster for backup", func() {
+			log.InfoD("Registering Source and Destination clusters and verifying the status")
+			ctx, err := backup.GetAdminCtxFromSecret()
+			log.FailOnError(err, "Fetching px-central-admin ctx")
+			err = CreateSourceAndDestClusters(orgID, "", "", ctx)
+			log.FailOnError(err, "Creating Source and destination cluster")
+			clusterStatus, _ = Inst().Backup.RegisterBackupCluster(orgID, SourceClusterName, "")
+			dash.VerifyFatal(clusterStatus, api.ClusterInfo_StatusInfo_Online, "Verifying backup cluster status")
+		})
+
+		//Create Schedule Backup
+		Step("Create Schedule Backup", func() {
+			ctx, err := backup.GetAdminCtxFromSecret()
+			log.FailOnError(err, "Fetching px-central-admin ctx")
+			backupName = fmt.Sprintf("%s-%v", BackupNamePrefix, time.Now().Unix())
+			timestamp := time.Now().Unix()
+			periodicPolicyName = fmt.Sprintf("%v-%v", "interval", timestamp)
+			log.InfoD("Creating backup interval schedule policy -%s", periodicPolicyName)
+			intervalSchedulePolicyInfo := Inst().Backup.CreateIntervalSchedulePolicy(5, 15, 2)
+
+			intervalPolicyStatus := Inst().Backup.BackupSchedulePolicy(periodicPolicyName, uuid.New(), orgID, intervalSchedulePolicyInfo)
+			log.InfoD("internalPolicyStatus - [%v]", intervalPolicyStatus)
+			dash.VerifyFatal(intervalPolicyStatus, nil, "Creating interval schedule policy")
+
+			log.InfoD("Fetching Schedule Backup uid")
```

Comment:
Providing `InfoD` is not necessary here. `Infof` is suffiicient. Also, provide the schedule backup name in the log statement.